tendrl/node_agent/objects/definition/node_agent.yaml

```diff
@@ -67,4 +67,52 @@ namespace.node_agent:
       list: nodes/$NodeContext.node.id/Networks
       help: "Global subnet wise network interface"
       value: /networks
+    ClusterAlertCounters:
+      enabled: True
+      attrs:
+        warning_count:
+          help: "Count of warning alerts for the cluster"
+          type: int
+        info_count:
+          help: "Count of ok alerts for the cluster"
+          type: int
+        integration_id:
+          help: "Id of the cluster"
+          type: String
+      value: /clusters/{0}/alert_counters
+      list: /clusters/{0}/alert_counters
+      help: "Cluster Alert Counter"
+    VolumeAlertCounters:
```

Comment:
Define and implement this object at https://github.com/Tendrl/node-agent/blob/master/tendrl/integrations/gluster/objects/definition/gluster.yaml

Remove from here